docs/OnnxReleases.md

```diff
@@ -45,13 +63,52 @@ repository=https://test.pypi.org/legacy/
 username=<username>
 password=<password>
 ```
-* Do ``twine upload dist/* -r pypitest`` to upload it to the test instance of pypi.
-* After uploading to pypitest, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
-* *NOTE - Once a distribution is uploaded to pypi, you cannot overwrite it on the same pypi instance.*
-* Once completely verified do, ``twine upload dist/*``  to upload to the official pypi.
-* *Conda - *
-    * Conda builds of ONNX are done via conda-forge, which runs infrastructure for building packages and uploading them to conda-forge. You need to submit a PR to https://github.com/conda-forge/onnx-feedstock (see https://github.com/conda-forge/onnx-feedstock/pull/1/files for an example PR.) You will need to have uploaded to PyPi already, and update the version number and tarball hash of the PyPi uploaded tarball.
-* The release branch needs to be marked as a protected branch to preserve it. The branch corresponding to each release must be preserved. (You may need to ask an admin to do this)
+* Do ``twine upload dist/* pypitest`` to upload it to the test instance of PyPI.
+* After uploading to PyPItest, you can test the source distribution by doing ``pip install --index-url https://test.pypi.org/legacy/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
+
+## PyPI package verification
+**Test ONNX itself**
+* Test the PyPI package installation with different combinations of various Python versions, Protobuf versions and platforms.
```

Comment:
Can you add the test matrix that we used for verification.

Python versions : Applicable python versions for the release
Protobuf versions : Latest protobuf version at the time of the release + protobuf version used for previous release
  | 3.5 | 3.6 | 3.7 | 3.8 |   |  
-- | -- | -- | -- | -- | -- | --
Linux |   |   |   |   |   |  
Windows |   |   |   |   |   |  
Mac |   |   |   |   |   |  